server/app/services/applicant/question/SingleSelectQuestion.java

```diff
@@ -32,6 +33,17 @@ public ImmutableList<Path> getAllPaths() {
     return ImmutableList.of(getSelectionPath());
   }
 
+  @Override
+  public ImmutableMap<Path, String> getJsonEntries() {
+    return ImmutableMap.of(
+        applicantQuestion.getContextualizedPath().join(Scalar.SELECTION),
+        applicantQuestion
+            .createSingleSelectQuestion()
+            .getSelectedOptionValue(LocalizedStrings.DEFAULT_LOCALE)
+            .map(LocalizedQuestionOption::optionText)
+            .orElse(""));
+  }
+
```

Comment:
Above code is taken from `case DROPDOWN`:

https://github.com/civiform/civiform/blob/358ad3dbb6e67aacb91afb3ede60e32ac281ba26/server/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java#L450-L508